packages/shared/src/components/modals/generic/common.ts

```diff
@@ -30,7 +30,7 @@ export const promotion: Record<string, MarketingCta> = {
       description:
         'Your saved posts are waiting for you on daily.dev mobile. Perfect for reading anytime, anywhere.',
       image: cloudinary.promotions.bookmarkLoops,
-      ctaText: 'Install the app',
+      ctaText: 'Tell me more',
```

Comment:
This is not in figma where was this change decided?

Reply:
This is for streaks. Kindly roll up the file to see the parent object.

Reply:
Oh, wait it should be the other object.